### ISSUE TITLE
WebViewPackageMissing crash on startup

### ISSUE THREAD

**Issue by mikehardy** [ankidroid/Anki-Android#6877]
###### Reproduction Steps

1. This one just keeps new ways to go down
2. 
3. 

###### Actual Result

```
java.lang.RuntimeException: 
  at android.app.ActivityThread.handleBindApplication (ActivityThread.java:6132)
  at java.lang.reflect.Method.invoke (Method.java)
  at com.swift.sandhook.SandHook.callOriginMethod (SandHook.java:185)
  at com.swift.sandhook.SandHook.callOriginMethod (SandHook.java:163)
  at com.swift.sandhook.xposedcompat.hookstub.HookStubManager.hookBridge (HookStubManager.java:304)
  at com.swift.sandhook.xposedcompat.hookstub.MethodHookerStubs32.stub_hook_0 (MethodHookerStubs32.java:198)
  at android.app.ActivityThread.access$1200 (ActivityThread.java:213)
  at android.app.ActivityThread$H.handleMessage (ActivityThread.java:1816)
  at android.os.Handler.dispatchMessage (Handler.java:106)
  at android.os.Looper.loop (Looper.java:215)
  at android.app.ActivityThread.main (ActivityThread.java:6939)
  at java.lang.reflect.Method.invoke (Method.java)
  at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run (RuntimeInit.java:493)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:870)
Caused by: android.util.AndroidRuntimeException: 
  at android.webkit.WebViewFactory.getProviderClass (WebViewFactory.java:471)
  at android.webkit.WebViewFactory.getProvider (WebViewFactory.java:251)
  at android.webkit.CookieManager.getInstance (CookieManager.java:40)
  at android.webkit.CookieManager.setAcceptFileSchemeCookies (CookieManager.java:267)
  at com.ichi2.anki.AnkiDroidApp.onCreate (AnkiDroidApp.java:271)
  at android.app.Instrumentation.callApplicationOnCreate (Instrumentation.java:1176)
  at android.app.ActivityThread.handleBindApplication (ActivityThread.java:6127)
  at java.lang.reflect.Method.invoke (Method.java)
  at com.swift.sandhook.SandHook.callOriginMethod (SandHook.java:185)
  at com.swift.sandhook.SandHook.callOriginMethod (SandHook.java:163)
  at com.swift.sandhook.xposedcompat.hookstub.HookStubManager.hookBridge (HookStubManager.java:304)
  at com.swift.sandhook.xposedcompat.hookstub.MethodHookerStubs32.stub_hook_0 (MethodHookerStubs32.java:198)
  at android.app.ActivityThread.access$1200 (ActivityThread.java:213)
  at android.app.ActivityThread$H.handleMessage (ActivityThread.java:1816)
  at android.os.Handler.dispatchMessage (Handler.java:106)
  at android.os.Looper.loop (Looper.java:215)
  at android.app.ActivityThread.main (ActivityThread.java:6939)
  at java.lang.reflect.Method.invoke (Method.java)
  at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run (RuntimeInit.java:493)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:870)
Caused by: android.webkit.WebViewFactory$MissingWebViewPackageException: 
  at android.webkit.WebViewFactory.getWebViewContextAndSetProvider (WebViewFactory.java:376)
  at android.webkit.WebViewFactory.getProviderClass (WebViewFactory.java:439)
  at android.webkit.WebViewFactory.getProvider (WebViewFactory.java:251)
  at android.webkit.CookieManager.getInstance (CookieManager.java:40)
  at android.webkit.CookieManager.setAcceptFileSchemeCookies (CookieManager.java:267)
  at com.ichi2.anki.AnkiDroidApp.onCreate (AnkiDroidApp.java:271)
  at android.app.Instrumentation.callApplicationOnCreate (Instrumentation.java:1176)
  at android.app.ActivityThread.handleBindApplication (ActivityThread.java:6127)
  at java.lang.reflect.Method.invoke (Method.java)
  at com.swift.sandhook.SandHook.callOriginMethod (SandHook.java:185)
  at com.swift.sandhook.SandHook.callOriginMethod (SandHook.java:163)
  at com.swift.sandhook.xposedcompat.hookstub.HookStubManager.hookBridge (HookStubManager.java:304)
  at com.swift.sandhook.xposedcompat.hookstub.MethodHookerStubs32.stub_hook_0 (MethodHookerStubs32.java:198)
  at android.app.ActivityThread.access$1200 (ActivityThread.java:213)
  at android.app.ActivityThread$H.handleMessage (ActivityThread.java:1816)
  at android.os.Handler.dispatchMessage (Handler.java:106)
  at android.os.Looper.loop (Looper.java:215)
  at android.app.ActivityThread.main (ActivityThread.java:6939)
  at java.lang.reflect.Method.invoke (Method.java)
  at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run (RuntimeInit.java:493)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:870)
```

###### Debug info
Refer to the [support page](https://ankidroid.org/docs/help.html) if you are unsure where to get the "debug info".

###### Research
*Enter an [x] character to confirm the points below:*

- [ ] I have read the [support page](https://ankidroid.org/docs/help.html) and am reporting a bug or enhancement request specific to AnkiDroid
- [ ] I have checked the [manual](https://ankidroid.org/docs/manual.html) and the [FAQ](https://github.com/ankidroid/Anki-Android/wiki/FAQ) and could not find a solution to my issue
- [ ] I have searched for similar existing issues here and on the user forum
- [ ] (Optional) I have confirmed the issue is not resolved in the latest alpha release ([instructions](https://docs.ankidroid.org/manual.html#betaTesting))

**Comment by david-allison-1** [comment]
#5794

**Comment by david-allison-1** [comment]
Fixed in #5794
